src/node_win32_etw_provider.cc

```diff
@@ -19,6 +19,7 @@
 // OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE
 // USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+#include "node_win32_etw_provider.h"
```

Comment:
Oddly, this does not work if I put `node_win32_etw_provider.h` behind `node_etw_provider.h`, but `node_etw_provider.h` is generated with a manifest and from what I can tell it only contains a bunch of constants..

Reply:
Which reminds me if the self-contain-ness can't be fixed properly then I will have to turn clang-format off here in https://github.com/nodejs/node/pull/16122 ...